contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -18,8 +19,26 @@ required to contribute
 * Read and review the link:doc_guidelines.adoc[Documentation guidelines] topic
 to understand the basic guidelines for consistency
 
+== Understanding the contribution workflow
+
+The following diagram outlines the steps required to add content to the OpenShift documentation:
+
+image::osdocs-contribution-workflow.png[OpenShift documentation contribution workflow]
+
+After you have identified a documentation requirement and created a ticket, you can contribute to the documentation directly or the OpenShift Docs team can create the content.
+
+When you contribute content directly, you must create a feature branch in a local clone of your own fork of the link:https://github.com/openshift/openshift-docs[openshift-docs repository]. After gathering stakeholder input and completing your technical testing, you can develop your documentation draft in your local feature branch. For more information about working with feature branches, see link:https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches[the GitHub documentation]. By using AsciiBinder on your local machine, you can create a local preview to review your updates.
+
+To submit your content for review, you must push your local feature branch to your openshift-docs fork on GitHub and create a PR. The PR is a request for the updates in your feature branch on your fork to be merged into the relevant branch in the mainstream openshift-docs repository. In the *Open a pull request* dialog, you can add a description, review the content updates, and create the PR. After your PR is created, the Netlify bot automatically generates a preview build and provides a preview link in a PR comment.
+
+The OpenShift Docs team reviews the PR and the Netlify preview build. The team also requests reviews from Quality Engineering (QE), subject matter experts (SMEs), and others, depending on the content that is submitted. You can apply any suggested changes by updating the original commit in your local feature branch. After you push the additional updates to your fork, the PR and the Netlify preview is updated automatically.
```

Comment:
is updated -> are updated (since there are two things)?

Reply:
@bergerhoffer thank you for your review and for highlighting this. Resolved.